packages/fa-icon-chooser/src/components/fa-icon-chooser/fa-icon-chooser.tsx

```diff
@@ -202,7 +202,16 @@ export class FaIconChooser {
   }
 
   getFamilies(): string[] {
-    return Object.keys(this.familyStyles);
+    // If we have kit metadata with embedProSvg permits, use those families
+    const embedProSvg = get(this, 'kitMetadata.permits.embedProSvg', []);
+
+    if (embedProSvg.length > 0) {
+      // Extract unique families from embedProSvg permits and sort alphabetically
+      const families = [...new Set(embedProSvg.map(permit => permit.family).filter(family => typeof family === 'string'))] as string[];
+      return families.sort();
+    }
+
+    return Object.keys(this.familyStyles).sort();
```

Comment:
will now default:
```
    classic: {
      solid: {
        prefix: 'fas',
      },
      regular: {
        prefix: 'far',
      },
      brands: {
        prefix: 'fab',
      },
    },
```

So even a free, pro-lite, and a pro+ lite  kit-token and api-token will show something.
No README.md update necessary.